docs/operate/customize/jobs.md

```diff
@@ -0,0 +1,141 @@
+---
+
+## 1. Define Your Celery Task
+
+Create your actual business logic as a Celery task:
+
+```python
+from celery import shared_task
+from your_package import get_records_with_expired_embargoes, lift_embargo, reindex_records
+
+@shared_task
+def update_expired_embargoes(since=None, **kwargs):
+    """Update and reindex records with expired embargoes."""
+    records = get_records_with_expired_embargoes(since)
+    for record in records:
+        lift_embargo(record)
+    reindex_records(records)
+```
+
+---
+
+## 2. Create a Job Class
+
+Use `JobType` to define a job that wraps your task. You can inherit directly or use the `create()` helper.
+
+```python
+from invenio_jobs.jobs import JobType
+
+class UpdateEmbargoesJob(JobType):
+    id = "update_expired_embargoes"
+    title = "Update expired embargoes"
+    description = "Lift and reindex expired embargoes."
+    task = update_expired_embargoes
+
+    @classmethod
+    def build_task_arguments(cls, job_obj, since=None, **kwargs):
+        return {"since": str(since)}
+```
+
+To include custom fields in the admin form, define a Marshmallow schema:
+
+```python
+from marshmallow import fields, validate
+from invenio_jobs.jobs import PredefinedArgsSchema
+
+class ExportEmbargoSchema(PredefinedArgsSchema):
+    dry_run = fields.Boolean(
+        required=False,
+        missing=False,
+        metadata={"description": "If true, no changes will be persisted."}
+    )
+
+class UpdateEmbargoesJob(JobType):
+    id = "update_expired_embargoes"
+    title = "Update expired embargoes"
+    description = "Lift and reindex expired embargoes."
+    task = update_expired_embargoes
+    arguments_schema = ExportEmbargoSchema
+
+    @classmethod
+    def build_task_arguments(cls, job_obj, since=None, dry_run=False, **kwargs):
+        return {
+            "since": str(since) if since else None,
+            "dry_run": dry_run
+        }
+```
+
+---
+
+## 3. Register Your Job and Task
+
+In your `setup.cfg`, register the task and job using entry points:
+
+```ini
+[options.entry_points]
+invenio_celery.tasks =
+    mysite_tasks = mysite.tasks
+
+invenio_jobs.jobs =
+    update_expired_embargoes = mysite.jobs:UpdateEmbargoesJob
+```
+!!! tip
+    **Note (local development only):** If you're developing locally and have modified `setup.cfg`, remember to re-run `pip install -e ./site` so that entry point changes take effect.
+
+---
+
+## 4. Run and Monitor Jobs
+
+Once your instance is restarted:
+
+- Navigate to the **Admin Panel → Jobs**.
+- Click **Create Job** and fill in the fields:
+    - Title and description.
+    - Select the task (e.g. `Update Expired Embargoes`).
+    - Choose the queue (a default queue should exist if jobs are enabled).
+- After saving, you can trigger the job immediately or schedule it.
+- You can also inspect previous runs and access detailed logs.
+
+---
+
+## Error Handling and Logging
```

Comment:
this section should be next to https://github.com/inveniosoftware/docs-invenio-rdm/pull/784/files#diff-f746076d7b34f08dba7552507cae91027d192ff10bd6118a49b7e97d5261e205R17, I think it is more relevant to mention it there, while creating the task to remember to log things

Reply:
I see your point, to me is more of an extra thing, but I agree it's more relevant there, I will move it as a subsection of it